simulation/halsim_ws_core/doc/wpilib-ws.yaml

```
@@ -0,0 +1,516 @@
+asyncapi: 2.0.0
+info:
+  title: WPILib WebSocket Remote Endpoint API
+  version: "1.0.0"
+  description: |
+    API to route WPILib HAL calls over WebSockets.
+  license:
+    name: MIT
```

```suggestion
    name: WPILib BSD
```
is this what the license is?

---

> is this what the license is?

The file is derived from @zhiquanyeo's version [here](https://github.com/bb-frc-workshops/wpilib-ws-schema/blob/master/schema/wpilib-ws.yaml). Assuming @zhiquanyeo doesn't mind changing the license, I suspect @CoolSpy3 would be OK with it.

---

Yup! Change the license if necessary :)